consumer-goods/4.0/modules/ROOT/pages/release-notes.adoc

```diff
@@ -0,0 +1,44 @@
+= Release Notes
+:cg-version: 4.0
+
+
+== v4.0
+2 June 2023
+
+=== Updates to the Trade Promotion effectiveness
+
+This release includes the https://anypoint.mulesoft.com/exchange/org.mule.examples/mulesoft-accelerator-for-consumergoods/minor/{cg-version}/pages/Use%20case%201b%20-%20Trade%20promotion%20effectiveness/[Trade promotion effectiveness^] use case, which helps visualize your promotions in one integrated platform with pre-built assets that integrate Business Intelligence Tools (BI Tools), such as CRM Analytics, into a TPM platform like Salesforce Consumer Goods Cloud.
```

Comment:
@nisha-nizamuddin I think we should add "updates to" in the use case description, otherwise it'll be the same description as the January 30th release.